var/spack/repos/builtin/packages/py-pandas/package.py

```diff
@@ -97,7 +97,7 @@ class PyPandas(PythonPackage):
     with default_args(type=("build", "run")):
         # Based on PyPI wheel versions
         depends_on("python@3.9:3.13", when="@2.2.3:")
-        depends_on("python@3.9:3.12", when="@2.1.1:")
+        depends_on("python@3.9:3.12", when="@2.1.1:2.2.2")
```

Comment:
I would suggest splitting again:

```python
        depends_on("python@3.9:", when="@2.1:")
        depends_on("python@3.8:", when="@1.4:")

        depends_on("python@:3.13")
        depends_on("python@:3.12", when="@:2.2.2")
        depends_on("python@:3.11", when="@:2.1.0")
        depends_on("python@:3.10", when="@:1.4")
        depends_on("python@:3.9", when="@:1.3.2")
        depends_on("python@:3.8", when="@:1.1.2")
```

Reply:
By "Based on PyPI wheel availability" I mean "Version 2.2.3 has wheels for Python 3.9–3.13"

Reply:
okay yeah, i thought the lowerbound would come from pyproject.toml